analyses/filter-mtp-tables/01-filter-mtp-tables-for-current-gencode.Rmd

```diff
@@ -83,8 +83,11 @@ rm(gencode_gtf)
 
 Get deprecated Open Targets Ensembl identifiers
 ```{r}
-# load OT deprecated ensembl IDs 
-ot_ensg_ids <- readr::read_lines(file.path(input_dir, "MTP_v11_InvalidENSG_20220831.txt"))
+# load MTP targets and diseases mapping file
+mtp_targets <- file.path(input_dir, "mtp-targets-mapping.tsv.gz")
+mtp_diseases <- file.path(input_dir, "mtp-diseases-mapping.tsv.gz")
+targets_mapping_file <- readr::read_tsv(file.path(input_dir, "mtp-targets-mapping.tsv.gz")) %>% as.data.frame() %>% tibble::as_tibble()
+disease_mapping_file <- readr::read_tsv(file.path(input_dir, "mtp-diseases-mapping.tsv.gz")) %>% as.data.frame() %>% tibble::as_tibble()
```

Comment:
Setting to read files twice. Kind of redundant, and the `mtp_targets` and `mtp_diseases` variables are not being utilized anywhere in the script.  Mapping files being read from the wrong location. They should not be in the module input folder but from the `mtp-annotations` module.